mmengine/model/base_model/data_preprocessor.py

```diff
@@ -158,13 +163,14 @@ class ImgDataPreprocessor(BaseDataPreprocessor):
     """
 
     def __init__(self,
+                 non_blocking: Optional[bool] = False,
```

Comment:
Add a new paramter in middle position may cause bc issue. Suggest moving it to the last.

Reply:
Done